articles/hooks/extensibility-points/send-phone-message.md

```diff
@@ -0,0 +1,124 @@
+@param {string} context.action - 'enrollment' or 'second-factor-authentication'
+@param {string} context.language - language used by login flow
+@param {string} context.code - one time password
+@param {string} context.ip - ip address
+@param {string} context.user_agent - user agent making the authentication request
+@param {object} context.client - object with details about the Auth0 application
+@param {string} context.client.client_id - Auth0 application ID
+@param {string} context.client.name - Auth0 application name
+@param {object} context.client.client_metadata - metadata from client
+@param {object} context.user - object representing the user
+@param {string} context.user.user_id - Auth0 user ID
+@param {string} context.user.name - user name
+@param {string} context.user.email - user email
+@param {object} context.user.app_metadata - metadata specific to user and application
+@param {object} context.user.user_metadata - metadata specific to user
+@param {function} cb - function (error, response)
+*/
+module.exports = function(recipient, text, context, cb) {
+ // TODO: Add your code here 
+  cb(null, {}); 
+};
+```
+
+::: note
+The callback function (`cb`) at the end of the sample code signals completion and **must** be included.
+:::
+
+## Example parameters
+
+This is an example of the parameters:
+
+```js
+{
+  "recipient": "1-808-555-5555",
+  "text": "Here is your one time password: 999111",
+  "context": {
+    "message_type": "sms",
+    "action": "enrollment",
+    "language": "en",
+    "code": "123456",
+    "ip": "127.0.0.1",
+    "user_agent": "Mozilla/5.0",
+    "client": {
+      "client_id": "1235",
+      "name": "Test Application",
+      "client_metadata": { }
+    },
+    "user": {
+      "user_id": "auth0|test12345",
+      "name": "Billie Magnusson",
+      "email": "billie@email.com",
+      "app_metadata": { },
+      "user_metadata": { }
+    }
+  }
+}
+```
+
+## Starter code response
+
+Once you have customized the Hook code, you can test it using the Runner embedded in the Editor. The Runner simulates a call to the Hook with the appropriate body and response.
+
+<%= include('../_includes/_test_runner_save_warning') %>
+
+When you run a Hook based on the starter code, the response object is:
+
+```
+{
+    "MessageID": "998a9ad1-c9b9-4b85-97b1-ac0305aa5532"
+}
+```
+
+## Localization
+
+The `context.language` parameter will always have one of the languages configured in the Tenant Settings (/universal-login/i18n). Depending on how you trigger the MFA flow, it will be calculated in different ways:
+
+- If you use the [MFA API](/mfa/concepts/mfa-api) it will use the Accept-Language header from the request, and map it to a tenant language. If the language is not available, it will be set to the tenant default language.
+
+- If you use the New Universal Login Experience, it will use a combination of the Accept-Language header and the `ui_locales` parameter, as described [here](/i18n#language-selection).
```

Comment:
```suggestion
- If you use the New Universal Login Experience, it will use a combination of the Accept-Language header and the `ui_locales` parameter, as described in [Universal Login Internationalization](/universal-login/i18n#language-selection).
```